Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://hosted.weblate.org/projects/jasp/jasp-desktop/) for JASP/JASP desktop.



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/jasp/-/jasp-desktop/horizontal-auto.svg)

Commit:
Translated using Weblate (Dutch)

Currently translated at 100.0% (4853 of 4853 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/nl/

vandenman commented on JASP-Desktop/po/jasp_nl.po:
```diff
@@ -20001,239 +20007,224 @@ msgctxt "Description|"
 msgctxt "LSestimationpredictions|"
 msgid "Number of future trials"
-msgstr ""
+msgstr "Aantal toekomstige proeven"
```
```suggestion
msgstr "Aantal toekomstige steekproeven"
```
of vertalen we dit in andere analyses ook zo?

FransMeerhoff replied:
Wordt door elkaar gebruikt. In desktop wordt meer 'proeven' gebruikt.
Consequent 'steekproef' lijkt mij dan beter.
@AlexanderLyNL wat vind jij?

AlexanderLyNL replied:
Toekomstige steekproefgrootte

vandenman commented on JASP-Desktop/po/jasp_nl.po:
```diff
@@ -20001,239 +20007,224 @@ msgctxt "Description|"
 msgctxt "LStestingpredictions|"
 msgid "Number of future trials"
-msgstr ""
+msgstr "Aantal toekomstige proeven"
```
zie andere comment

vandenman reviewed (changes requested):
I'm not sure about whether `trials` should be translated as `proeven`. I think that either `steekproeven` or `observaties` would be better.

JoKeyser committed:
Translated using Weblate (German)

Currently translated at 82.4% (1776 of 2154 strings)

Translation: JASP/JASP engine
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-engine/de/

Commit:
Translated using Weblate (Dutch)

Currently translated at 100.0% (4853 of 4853 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/nl/

Commit:
Translated using Weblate (Dutch)

Currently translated at 99.3% (2141 of 2154 strings)

Translation: JASP/JASP engine
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-engine/nl/

Commit:
Translated using Weblate (Dutch)

Currently translated at 100.0% (4853 of 4853 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/nl/

Commit:
Translated using Weblate (Dutch)

Currently translated at 100.0% (4853 of 4853 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/nl/

Commit:
Translated using Weblate (Dutch)

Currently translated at 99.3% (2141 of 2154 strings)

Translation: JASP/JASP engine
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-engine/nl/

Commit:
Translated using Weblate (German)

Currently translated at 97.8% (4750 of 4853 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/de/

boutinb committed:
Merge branch 'stable' into weblate-jasp-jasp-desktop
